Sources/DeepLearning/Optimizer.swift

```diff
@@ -91,11 +112,24 @@ public class Adam<Model: Layer, Scalar: TensorFlowFloatingPoint>: Optimizer
     }
 }
 
+/// RMSProp optimizer.
+///
+/// It is recommended to leave the parameters of this optimizer at their default values (except the
+/// learning rate, which can be freely tuned). This optimizer is usually a good choice for recurrent
+/// neural networks.
+///
+/// References:
+/// - [rmsprop: Divide the gradient by a running average of its recent magnitude](
```

Comment:
Is there a "references" syntax?

Reply:
There isn't. It ends up looking like this:
![image](https://user-images.githubusercontent.com/5590046/53789081-7519e480-3ed8-11e9-8ac9-92a0b61392be.png)

There isn't an appropriate comment type, here's the full list: https://github.com/apple/swift/blob/master/include/swift/Markup/SimpleFields.def

Let me know if you'd like any changes, otherwise I'll keep things as is.

Reply:
The issue I worry about is that the bullet is not being indented under "References:" in your rendered text.

Reply:
Yeah, that's unfortunate. Indenting the hyphen further right doesn't help.

Reply:
Can you drop the hyphen and put the reference right after the colon? Also make "references" be singular.

Reply:
Done in https://github.com/tensorflow/swift-apis/pull/46/commits/14de85f78cf2977697e1b470b9becc4b084eabc4.

- https://tensorflow.devsite.corp.google.com/swift/api_docs/Classes#adam
![image](https://user-images.githubusercontent.com/5590046/53791838-30de1280-3edf-11e9-803b-24c8dba581f7.png)

- https://tensorflow.devsite.corp.google.com/swift/api_docs/Classes/Adam
![image](https://user-images.githubusercontent.com/5590046/53791847-35a2c680-3edf-11e9-8d41-2b23782b2fbc.png)

Reply:
I'm not adding this bullet. Any convention to follow here? Seems it's best to keep it flat.